client/Lykke.Service.PayInternal.Client/Api/IMerchantsApi.cs

```diff
@@ -39,5 +42,26 @@ internal interface IMerchantsApi
 
         [Get("/api/merchants/{merchantId}/markups/{assetPairId}")]
         Task<MarkupResponse> ResolveMarkupAsync(string merchantId, string assetPairId);
+
+        #endregion
+
+        #region merchant groups
+
+        [Post("/api/merchantGroups")]
+        Task<MerchantGroupResponse> AddGroupAsync([Body] AddMerchantGroupRequest request);
+
+        [Get("/api/merchantGroups/{id}")]
+        Task<MerchantGroupResponse> GetGroupAsync(string id);
+
+        [Put("/api/merchantGroups")]
```

Comment:
How do I know the Id of MerchantGroup - need method to get MerchantGroups by OwnerId